.travis.yml

```diff
@@ -10,7 +10,7 @@ cache:
     - "node_modules"
 deploy:
   provider: azure_web_apps
-  on: develop
+  on: feature/5/azure
```

Comment:
is this correct?

Reply:
Nope, was for testing purposes

Reply:
If you want to deploy manually from time to time, I can also give you the git credentials.